CHANGELOG.md

```diff
@@ -6,7 +6,18 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-
++ ### Addedd
++ - GA4 new event names:
++ - - viewCart
++ - - beginCheckout
++ - - addPaymentInfo
++ - - addShippingInfo
++ - - addToWishlist
++ - - share
++ - - login
++ - - search
++ - - signup
++ - - refund
```

Comment:
The markdown formatting got weird [here](https://github.com/vtex-apps/pixel-manager/blob/feat/ga4-new-event-names/CHANGELOG.md):
![CleanShot 2023-04-03 at 16 59 43](https://user-images.githubusercontent.com/381395/229614954-6538aa2e-7de5-438b-ab9d-12f8e54b1b7a.png)

See if that helps:
```suggestion

### Added
- GA4 new event names:
  - `viewCart`
  - `beginCheckout`
  - `addPaymentInfo`
  - `addShippingInfo`
  - `addToWishlist`
  - `share`
  - `login`
  - `search`
  - `signup`
  - `refund`

```
![CleanShot 2023-04-03 at 17 06 38](https://user-images.githubusercontent.com/381395/229615698-3e1bc5d0-d852-43e0-ba01-24771bfd6c92.png)